client/layout/masterbar/notifications.jsx

```diff
@@ -134,7 +134,7 @@ class MasterbarItemNotifications extends Component {
 					/>
 				</MasterbarItem>
 				<AsyncLoad
-					require="../../../apps/notifications/index.jsx"
+					require="@automattic/notifications/index.jsx"
```

Comment:
Looks like the should be extracted to a package? It is weird to have an app (`calypso`) depending on other app (`@automattic/notifications`)

Reply:
Definitely, this is one of the areas where the `notifications` package needs some love. Some time ago, it was imported into the monorepo from a previously separate repo, and we did only the minimal changes needed to make it work.

There should be a `packages/notifications` package, used by Calypso app and also by a standalone Notifications app used e.g. in masterbar of site frontends. And an `apps/notifications` app, for that standalone app, a very thin wrapper around the package.